waspc/cli/Command/Deps.hs

```diff
@@ -0,0 +1,27 @@
+module Command.Deps
+  ( deps,
+  )
+where
+
+import Cli.Terminal (title)
+import Command (Command)
+import Control.Monad.IO.Class (liftIO)
+import qualified Generator.ServerGenerator as ServerGenerator
+import qualified Generator.WebAppGenerator as WebAppGenerator
+import NpmDependency (printDep)
+
+deps :: Command ()
+deps =
```

Comment:
This comment: https://github.com/wasp-lang/wasp/pull/238#discussion_r634287077 .

Reply:
Left a comment, Ormolu formats it like that!

Reply:
Oh wow, ok I did not expect that! Btw one way you could go around this is then to use `liftIO . putStrLn . unlines $`. But it doesn't really matter, it is also ok like this.